demos/share-files.html

```diff
@@ -128,8 +128,16 @@ <h1>Web Share Test</h1>
           testWebShareDelay);
 
       if (navigator.share === undefined) {
-        logError('Error: You need to use a browser that supports this draft ' +
-                 'proposal.');
+        if (window.location.protocol === 'http:') {
+          logError('navigator.share() is only available in secure contexts.');
+          setTimeout(() => {
+            window.location.replace(window.location.href.replace(/^http:/, 'https:'));
+          }, 4000);
```

Comment:
4 seconds seems like a long time to wait for the redirect? Should we just make it happen straight away?